### ISSUE TITLE
Classic Chaos Run

### ISSUE THREAD

**Issue by chojo31** [kolton/d2bot-with-kolbot#1020]
Hi,
I would like to know if there is a script that run the Chaos Sanctuary with a Tele Sorc / Hdin combo.

I would like the Sorc to only tele to star, then make a tp and idle in town. She will quit when my hdin killed diablo.
The job of my hdin is to take my sorc tp and clear Chaos Sanctuary alone.

Thank you in advance to let me know if there is a way to accomplish that.

**Comment by KolbotMod** [comment]
send me a folder with a bot I will see

**Comment by 5noop** [comment]
> send me a folder with a bot I will see

Do **not** do that because:
1) It won't help
2) Your passwords are in some files in your bot folder

**Comment by 5noop** [comment]
I actually made this script a while ago https://pastebin.com/76DkQPb0
Basically the sorc opens each seals and tp at star in the end. Now if you want her to idle in town just put a `delay(1200000);` in the end before the `return true;`. And if you want her to open every seal (she let one open atm), just uncomment this line `//this.openSeal(395);`.

You can simply run diablohelper on your pala and it should work. But without barb I'd say it's probably not the best compo in classic. With 2 keys i'd rather run barb + sorc or even barb + hdin.

**Comment by noah-** [comment]
You can try this: https://gist.github.com/noah-/2685fbeccc72fd595bbe89116aea272e

**Comment by chojo31** [comment]
> 
> 
> I actually made this script a while ago https://pastebin.com/76DkQPb0
> Basically the sorc opens each seals and tp at star in the end. Now if you want her to idle in town just put a `delay(1200000);` in the end before the `return true;`. And if you want her to open every seal (there is one she won't touch atm), just uncomment this line `//this.openSeal(395);`.
> 
> You can simply run diablohelper on your pala and it should work. But without barb I'd say it's probably not the best compo in classic. With 2 keys i'd rather run barb + sorc or even barb + hdin.

Thank you!
What are the needed config lines to add to each of my sorc and hdin config?

**Comment by 5noop** [comment]
Yeah you shoud add `Config.SealsOpener.WaitBO = false; // Wait for BO at wp` in your sorc file, and the equivalent in Config.js
